__tests__/app.test.js

```diff
@@ -81,3 +81,46 @@ describe("/api/articles/:article_id", () => {
     });
   });
 });
+
+describe("/api/articles", () => {
+  describe("GET", () => {
+    test("STATUS 200 - Responds with an array of all the article objects sorted by the date in descending order", () => {
+      return request(app)
+        .get("/api/articles?sort_by=created_at")
```

Comment:
The sort should be default, you're getting a little ahead of yourself on the endpoint here.